src/main/resources/mqtt2influx-configuration.yaml

```diff
@@ -24,3 +24,20 @@ devices:
       Total_in: Total Consumption
       Total_out: Total Production
       Power_curr: Current Consumption
+  - name: solar
+    type: raw
+    topic: solar/114183239002/0/#
```

Comment:
Please change the ID to something different, as it is the number of my parents

Reply:
This is the topic and we are using it to fetch the data properly. Can you change the device topic in the MQTT in this case?